kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionSidePanel.vue

```diff
@@ -4,6 +4,7 @@
     v-if="$route.params.section_id"
     ref="resourcePanel"
     alignment="right"
+    sidePanelWidth="700px"
```

Comment:
While this is probably the width that is listed on the spec, it will be too big on smaller screens. This should be updated so that the width is 700px on larger screens, and smaller on small screens. In other places, we have a full-screen panel on mobile view, for example, where the width is 100% of the screen width. Worth confirming with @tomiwaoLE that this is the approach that would be correct for the designs, and also checking with @nucleogenesis about to implement (and if there is any logic we can repurpose from other places where we solve the same problem)

Reply:
Hi @tomiwaoLE ,I am just confirming to find out the intended side panel behaviour on a mobile device, could it be that it should occupy half the screen size or can take full screen ?

Reply:
@AllanOXDi IIRC the side panel goes full width on mobile - you can double check by opening it for a piece of content in the library